test/intfuncs.jl

```diff
@@ -487,3 +487,15 @@ end
     ci = code_typed(() -> 14 // 21)[][1]
     @test ci.code == Any[Core.ReturnNode(2 // 3)]
 end
+
+@testset "binomial" begin
+    for T in (Bool, Int8, UInt16, Int32, UInt64)
+        for x in rand(T, 1000)
```

Comment:
I think this is probably a little excessive 😂

Reply:
It's still 100% compilation time...

Reply:
I guess it won't hurt